### DISCUSSION TITLE
How to read the data column of the Audit information, exported column data as [object Object] in the auditlog.csv file

### DISCUSSION THREAD

**Discussion by yevon** [globaleaks/globaleaks-whistleblowing-software#4246] [Q&A]
It seems that Globaleaks does store an object containing more detailed data about the Audit row, but when using the export file in the auditlog.csv file, it gets exported as [object Object] and cannot be read. It would be nice that this information could get exported as a JSON column in the csv file. Is there a another way to read the information of the audit log, including the data column?

**Comment by evilaliv3** [comment]
Thank you yevon, we will try to correct this within the next release exporing the object correctly (that will be a json string)

**Reply by yevon** [reply]
Nice thanks!
